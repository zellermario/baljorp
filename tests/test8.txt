create SJ m 2

create Eszkimo e m
create Lapat l player e

digobject e
useobject l e
pass e