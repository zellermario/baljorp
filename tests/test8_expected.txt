Fields:
SJ m (0) - object: l	neighbours: []

Buildings:

Actors:
> Eszkimo e (heat: 5) on m
  Inventory: []